heare/developer/context.py

```diff
@@ -289,3 +320,66 @@ def flush(self, chat_history, compact=True):
         # Write the data to the file
         with open(history_file, "w") as f:
             json.dump(context_data, f, indent=2, cls=PydanticJSONEncoder)
+
+
+def load_session_data(session_id: str) -> Tuple[list, list, Any, Optional[str]]:
```

Comment:
this should return an instance of AgentContext. figure out what else it needs to do that (likely either an existing agent context or at least an instance UserInterface. if we're ever dropping info on the floor, make sure to flush the agent context to disk first.